lib/vmpooler/pool_manager.rb

```diff
@@ -555,6 +581,17 @@ def _check_pool(pool)
         end
       end
 
+      # MIGRATIONS
+      $redis.smembers('vmpooler__migrating__' + pool['name']).each do |vm|
+        if inventory[vm]
+          begin
+            migrate_vm(vm, pool['name'])
+          rescue
+            $logger.log('s', '[x] [' + pool['name'] + "] '" + vm + "' failed to migrate")
+          end
+        end
+      end
+
```

Comment:
What hypervisor data are you hoping to see? The only thing that stands out to me is if you wanted to know the host your VM is running on we could report that when logging that no migration is required. This will already be reported when a migration is performed.